themes/ddbasic/gulpfile.js

```diff
@@ -45,7 +45,7 @@ gulp.task('sass', 'Process SCSS using libsass',
         includePaths: [
           'node_modules/compass-mixins/lib',
           // Zen grids is downloaded as a library using drush make.
-          '../../libraries/zen-grids/stylesheets'
+          './zen-grids/stylesheets'
```

Comment:
Why this change?

As I see it [zen-grids](https://github.com/ding2/ding2/blob/master/ding2.make#L392) is not placed in the theme folder but as a library.